django/template/loaders/filesystem.py

```diff
@@ -14,6 +15,18 @@
 from .base import Loader as BaseLoader
 
 
+class FileSystemOrigin(Origin):
+    def __init__(self, *args, mtime=None, **kwargs):
```

Comment:
The `mtime` kwarg breaks Py2. Is it even used? The only use I can find of direct assignment of `FileSystemOrigin.mtime`.